1228-public_page/user_navigates_to_landing_page_for_registration_and_can_see_files.feature

```diff
@@ -0,0 +1,51 @@
+Feature: User navigates to Landing Page for Registration and can see Files
+
+    @1530
+    Scenario: Files that are Administrative Agreements are hidden
+        Given Anonymous User views Landing Page for Registration
+        And the Registration contains a File, which is an Administrative Agreement
+        When they view the Files section
+        Then they do not see the File that is an Administrative Agreement
+
+    Scenario: Files that are part of Registration are listed
+        Given Anonymous User views Landing Page for Registration
+        And the Registration contains Files
+        When they view the Files section
+        Then they can see Files that are not Administrative Agreements are listed
+        And for each file they can see:
```

Comment:
```suggestion
        And for each File they can see:
```

Reply:
herregud, hvor blind kan man være uten å faktisk være blind.... valgte selvsagt å søke på "files" i stedet for "file" 🙃